frontend/lib/src/components/widgets/DateInput/DateInput.tsx

```diff
@@ -408,6 +407,9 @@ class DateInput extends React.PureComponent<Props, State> {
                       paddingBottom: ".5rem",
                       paddingTop: ".5rem",
                     },
+                    props: {
+                      "data-testid": "stDateInputInput",
```

Comment:
My bad for oversight in my proposal on semantically named override sections like this - thoughts on `stDateInput-Input` instead?

Reply:
haha not a problem. Just trying to stick with the proposal. I'm totally okay with 
`stDateInput-Input`.

Should I add a section into the proposal for things like this? I think I sometimes see odd one offs.

Reply:
That would be great - definitely worth adding as we encounter these scenarios!

Reply:
Great will add in a bit.